cmake/helpers/configure.cmake

```diff
@@ -397,6 +398,39 @@ else ()
     HAVE_SHARED_MUTEX
   )
 
+  # std::atomic<int64_t> requires linking against -latomic on some esoteric platforms
```

Comment:
Perhaps "many 32-bit CPUs" would give the reader a hint in the right direction, vs esoteric.

Reply:
rephrased